### DISCUSSION TITLE
Drag'n'drop list inside Mantine's modal

### DISCUSSION THREAD

**Discussion by crabvk** [hello-pangea/dnd#799] [General]
Hi. I put dnd list inside a [Mantine's modal](https://mantine.dev/core/modal/) and have an issue: when I drag a list item it appears far right (offset equal to modal left position, I guess). Here's the example: https://stackblitz.com/~/github.com/crabvk/mantine-modal-dnd
Could you please suggest a solution on how to fix this?

**Comment by AlinaTokareva** [comment]
Hello @crabvk! It wasn't easy, but I've found the answer. You should use **renderClone** in <Droppable>. See [official docs](https://github.com/hello-pangea/dnd/blob/main/docs/guides/reparenting.md)
